src/csharp/build/common.props

```diff
@@ -19,11 +19,8 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
   </PropertyGroup>
 
-  <!-- Needed for the net45 build to work on Unix. See https://github.com/dotnet/designs/pull/33 -->
-  <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
-  </ItemGroup>
+  <PropertyGroup>
+    <LangVersion>7.2</LangVersion>
```

Comment:
I think setting this to `Latest` is a better option.

Reply:
I think 7.2 is enough for now and it allows the tests to be more deterministic as we have different version of dotnet SDK on different platforms on our CI. I don't want to our LangVersion to change if we decide to upgrade dotnet SDK while performing maintenance of our testing infrastructure.

Reply:
Hmm, So there's no plan to add support for some features which exist in newer versions of C# like `null ref types` at least for now?

Reply:
Needs for other language features can be evaluated separately. For now we want C# 7.2 and I explained before why setting langversion explicitly makes sense.